spec/factories/users.rb

```diff
@@ -15,4 +24,23 @@
     roles { ROLE::CERTIFICATE_MANAGER }
     permissions { UserRolePermissions.new(ROLE::CERTIFICATE_MANAGER, nil) }
   end
+
+  factory :user_manager_user, class: User do
+    given_name { "Jane" }
+    family_name  { "Smith" }
+    email { "test@test.test" }
+    password { "validpassword" }
+    roles { ROLE::USER_MANAGER }
+    permissions { UserRolePermissions.new(ROLE::USER_MANAGER, nil) }
+  end
+
+  factory :component_manager_user, class: User do
+    given_name { "Jane" }
+    family_name  { "Smith" }
+    email { "test@test.test" }
+    password { "validpassword" }
+    roles { ROLE::COMPONENT_MANAGER }
+    permissions { UserRolePermissions.new(ROLE::COMPONENT_MANAGER, nil) }
+  end
```

Comment:
If the role is the only thing that changes we can just have one user factory and overwrite the role each time we need to